pkgs/by-name/sy/sysdig-cli-scanner/package.nix

```diff
@@ -2,54 +2,71 @@
-  version = versionMetadata.version;
-
-  src = fetchurl { inherit (fetchForSystem) url hash; };
 
-  nativeBuildInputs = [ makeWrapper ];
+  sysdig-cli-scanner-unwrapped = stdenv.mkDerivation {
+    pname = "sysdig-cli-scanner-unwrapped";
+    version = versionMetadata.version;
 
-  dontUnpack = true;
+    src = fetchurl { inherit (fetchForSystem) url hash; };
 
-  installPhase = ''
-    runHook preInstall
+    dontUnpack = true;
 
-    install -Dm755 -T $src $out/bin/sysdig-cli-scanner
+    installPhase = ''
+      runHook preInstall
 
-    wrapProgram $out/bin/sysdig-cli-scanner \
-      --add-flags --dbpath="\$HOME/.cache/sysdig-cli-scanner/"
+      install -Dm755 -T $src $out/bin/sysdig-cli-scanner
 
-    runHook postInstall
-  '';
+      runHook postInstall
+    '';
 
-  passthru.updateScript = ./update.sh;
+    passthru.updateScript = ./update.sh;
 
-  meta = with lib; {
-    description = "Tool for scanning container images and directories using Sysdig";
-    longDescription = ''
-      The Sysdig Vulnerability CLI Scanner, sysdig-cli-scanner, is a versatile tool designed to
-      manually scan container images and directories, whether they are located locally or remotely.
-      Depending on your specific use case, you have the flexibility to execute sysdig-cli-scanner
-      in Vulnerability Management (VM) mode for image scanning or Infrastructure as Code (IaC) mode
-      for scanning directories.
-    '';
-    homepage = "https://docs.sysdig.com/en/docs/installation/sysdig-secure/install-vulnerability-cli-scanner/";
-    mainProgram = "sysdig-cli-scanner";
-    license = licenses.unfreeRedistributable;
-    maintainers = with maintainers; [ tembleking ];
-    platforms = [
-      "x86_64-linux"
-      "aarch64-linux"
-      "x86_64-darwin"
-      "aarch64-darwin"
-    ];
-    sourceProvenance = with sourceTypes; [ binaryNativeCode ];
+    meta = with lib; {
+      description = "Tool for scanning container images and directories using Sysdig";
+      longDescription = ''
+        The Sysdig Vulnerability CLI Scanner, sysdig-cli-scanner, is a versatile tool designed to
+        manually scan container images and directories, whether they are located locally or remotely.
+        Depending on your specific use case, you have the flexibility to execute sysdig-cli-scanner
+        in Vulnerability Management (VM) mode for image scanning or Infrastructure as Code (IaC) mode
+        for scanning directories.
+      '';
+      homepage = "https://docs.sysdig.com/en/docs/installation/sysdig-secure/install-vulnerability-cli-scanner/";
+      mainProgram = "sysdig-cli-scanner";
+      license = licenses.unfreeRedistributable;
+      maintainers = with maintainers; [ tembleking ];
+      platforms = [
+        "x86_64-linux"
+        "aarch64-linux"
+        "x86_64-darwin"
+        "aarch64-darwin"
+      ];
+      sourceProvenance = with sourceTypes; [ binaryNativeCode ];
+    };
   };
+in
+(writeShellApplication {
```

Comment:
We don't need to use an extra derivation here. We can replace the old wrapper with this.

Reply:
I don't really know how to do it, can you show me an example please?

Reply:
Hi @SuperSandro2000, can you help me with this please?